src/init.cpp

```diff
@@ -788,7 +788,9 @@ bool AppInit2(boost::thread_group& threadGroup)
     LogPrintf("Using at most %i connections (%i file descriptors available)\n", nMaxConnections, nFD);
     std::ostringstream strErrors;
 
-    LogPrintf("Using %u threads for script verification\n", nScriptCheckThreads);
+    // nScriptCheckThreads == 0 means we want to display it as 1 thread
+    unsigned int logScriptCheckThreads = (nScriptCheckThreads == 0) ? 1 : nScriptCheckThreads;
+    LogPrintf("Using %u threads for script verification\n", logScriptCheckThreads);
```

Comment:
Yes, "0 threads" means that verification is done in the caller thread. But I don't think it's worth changing this code for, sorry. We're already overworked, let's try to keep down on the number of unnecessary changes.